simphony_metadata.yml

```diff
@@ -16,7 +16,8 @@ CUDS_KEYS:
     parent:
     CUBA.UUID:
       scope: CUBA.SYSTEM
-    CUBA.DATA:
+    data:
```

Comment:
Isn't `data` a fixed metadata attributes?  CUBA.DATA is no primary type (but a container) that I think we would have problem defining it in cuba.yml (or any suggestion?)